src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -480,6 +481,58 @@ public void ascii() throws IOException {
         );
     }
 
+    @Test
+    public void ifWithThreeParams() throws IOException {
+        assertThat(
+                executeQuery("SELECT if(age > 30, 'True', 'False') AS Ages FROM " + TEST_INDEX_ACCOUNT
+                        + " WHERE age IS NOT NULL GROUP BY Ages", "jdbc"),
+                containsString("\"type\": \"keyword\"")
+        );
+        assertThat(
+                executeQuery("SELECT if(2 > 0, 'hello', 'world') AS iif FROM " + TEST_INDEX_ACCOUNT),
+                hitAny(kvString("/fields/iif/0", equalTo("hello")))
+        );
+        assertThat(
+                getHits(
+                executeQuery("SELECT iif(2 = 0, 'hello') AS iif FROM " + TEST_INDEX_ACCOUNT)
+                ).getJSONObject(0).query("/fields/iif/0"),
+                equalTo(null)
+        );
+    }
+
+    @Test
+    public void ifWithTwoParams() throws IOException {
+        assertThat(
+                executeQuery("SELECT if(age > 30, 'True') AS Ages FROM " + TEST_INDEX_ACCOUNT
+                        + " WHERE age IS NOT NULL GROUP BY Ages", "jdbc"),
+                containsString("\"type\": \"keyword\"")
+        );
+
+        JSONObject response = executeQuery("SELECT iif(2 = 0, 'hello') AS iif FROM " + TEST_INDEX_ACCOUNT);
+        JSONObject hit = getHits(response).getJSONObject(0);
+        assertThat(
+                ((JSONArray) hit.query("/fields/iif")).get(0),
+                equalTo(null)
+        );
+    }
+
+    @Test
+    public void ifnull() throws IOException {
+        assertThat(
+                executeQuery("SELECT IFNULL(lastname, 'unknown') AS name FROM " + TEST_INDEX_ACCOUNT
+                        + " GROUP BY name", "jdbc"),
+                containsString("\"type\": \"keyword\"")
+        );
+        assertThat(
+                executeQuery("SELECT IFNULL('sample', 'IsNull') AS name FROM " + TEST_INDEX_ACCOUNT),
```

Comment:
Could you add the test for NULL value?
1) SELECT IFNULL(NULL,10); expected 10
https://dev.mysql.com/doc/refman/8.0/en/control-flow-functions.html#function_ifnull